packages/storybook-addon-performance/register.mjs

```diff
@@ -0,0 +1,2 @@
+// Separate entry point for storybook addon registration
+import './dist/mjs/register';
```

Comment:
Where does this path get generated?

Reply:
Line 4: "outDir": "dist/esm" of tsconfig.json inside  packages/storybook-addon-performance.

Reply:
When you run typescript:build two directories are created inside dist: esm and cjs.

Reply:
This path says `mjs` tho is that a typo?

Reply:
You're right.

Reply:
Fixed it. 👍🏽